Dockerfiles/agent/Dockerfile

```diff
@@ -115,6 +120,11 @@ RUN tar xzf s6.tgz \
  && chown root:root /readsecret.py \
  && chmod 500 /readsecret.py
 
+# Update links to python binaries
+RUN ln -sfn /opt/datadog-agent/embedded/bin/python${PYTHON_VERSION}.7 /opt/datadog-agent/embedded/bin/python \
```

Comment:
the minor version could change (especially for python3). Aliases named `python2` and `python3` should be available.

We could simply do:
```
ln -sfn /opt/datadog-agent/embedded/bin/python${PYTHON_VERSION} /opt/datadog-agent/embedded/bin/python \
ln -sfn /opt/datadog-agent/embedded/bin/python${PYTHON_VERSION}-config /opt/datadog-agent/embedded/bin/python-config
```

Reply:
Yeah that's a better form. With the same reasoning I guess it would also be better to turn 
```
 && if [ "$PYTHON_VERSION" = "2" ]; then rm -rf opt/datadog-agent/embedded/lib/python3.7; fi \
 && if [ "$PYTHON_VERSION" = "3" ]; then rm -rf opt/datadog-agent/embedded/lib/python2.7; fi \
```

into

```
 && if [ "$PYTHON_VERSION" = "2" ]; then rm -rf opt/datadog-agent/embedded/lib/python3*; fi \
 && if [ "$PYTHON_VERSION" = "3" ]; then rm -rf opt/datadog-agent/embedded/lib/python2*; fi \
```